dask_expr/_accessor.py

```diff
@@ -57,19 +57,17 @@ def _function_map(self, attr, *args, **kwargs):
 
         if isinstance(self._series, Index):
             return new_collection(
-                FunctionMapIndex(
-                    self._series.expr, self._accessor_name, attr, args, kwargs
-                )
```

Comment:
It's nice seeing all of this `.expr` complexity going away

Reply:
Yep 100% agree